### ISSUE TITLE
Why is there no Console Tab in Run? 

### ISSUE THREAD

**Issue by xjf1128** [flutter/flutter#25835]
Why is there no Console Tab in Run?It can still be shown the other day. I've tried to restart the computer.
Android Studio: 3.2.1
<img width="1384" alt="1111" src="https://user-images.githubusercontent.com/17099561/50507103-f909b780-0ab6-11e9-8f83-13d6681615fd.png">

**Comment by xjf1128** [comment]
It can be compiled and installed, but there is no console information.

**Comment by zoechi** [comment]
That's likely a pure AndroidStudio issue. Try deleting the AndroidStudio settings folder (or just rename to not lose it) and then start AS again and the tab will be back.
I don't know if there is a way to get it back without dropping all settings.

**Comment by xjf1128** [comment]
> That's likely a pure AndroidStudio issue. Try deleting the AndroidStudio settings folder (or just rename to not lose it) and then start AS again and the tab will be back.
> I don't know if there is a way to get it back without dropping all settings.

But I run the Andorid native project to display the running information.And I don't want to reset Android Studio.

**Comment by zoechi** [comment]
> And I don't want to reset Android Studio.

Just check if it solves the issue. You can restore the settings afterwards. 
We can't provide support for Android Studio issues unrelated to Flutter.

**Comment by hash404** [comment]
Have the same issue, when I upgraded Flutter  (on master) the Console tab went missing!

**Comment by zoechi** [comment]
@hash404 and when you switch back to another channel?

**Comment by hash404** [comment]
Still missing when I switch to `flutter channel stable`.

The Logcat appears when I open an Android app in Android Studio, but when I open a Flutter app in Android Studio there is no option for Logcat.

**Comment by zoechi** [comment]
Did you try removing/renaming the directory that contains your settings?

**Comment by y-polek** [comment]
I was able to get Console tab back with "Restore Layout" action:
1. Focus "Run" tab (click on tab header, not on its body)
2. Ctrl+Shift+A, find and click "Restore Layout"

**Comment by zoechi** [comment]
@xjf1128 does this work for you as well?

**Comment by xjf1128** [comment]
> I was able to get Console tab back with "Restore Layout" action:
> 
> 1. Focus "Run" tab (click on tab header, not on its body)
> 2. Ctrl+Shift+A, find and click "Restore Layout"

Thank you very much for your help. I solved this problem.

**Comment by yurkoaapelsinn** [comment]
You can just make some compilation errors (for example add  const String a = 1234;)  run the application and console will reveal. That worked for me.

**Comment by zaai123** [comment]
https://github.com/flutter/flutter-intellij/issues/3673#issuecomment-517356243

**Comment by tarek360** [comment]
**Solution:**
- Close Android Studio.
- Go to `/Library/Preferences/AndroidStudio3.5/options`
- Delete `runner.layout.xml`
- Start Android Studio.

Enjoy!!

**Comment by adriancmurray** [comment]
I found the "Flutter Attach" process was the culprit when this happened to me. Restarting the app or resetting the layout didn't work and I didn't really want to delete files. I tried the Flutter Attach process and then stopped all processes. Once I did that, it was back to normal. Hope that helps someone.

<img width="140" alt="Screen Shot 2019-09-28 at 2 02 35 AM" src="https://user-images.githubusercontent.com/39349880/65813544-1bd71280-e194-11e9-8bb2-d2bea909d3b0.png">
